src/qmapshack/poi/CPoiPOI_TagMap.cpp

```diff
@@ -0,0 +1,28 @@
+/**********************************************************************************************
+    Copyright (C) 2021 Henri Hornburg <pingurus@t-online.de>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+**********************************************************************************************/
+
+#include "CPoiPOI.h"
+
+QMap<QString, QString> CPoiPOI::tagMap;
+QMap<QString, QString> CPoiPOI::initTagMap()
+{
+    QMap<QString, QString> map;
+    map["accommodation_hotel"] = "tourism=hotel";
+    map["water_weir"] = "waterway=weir";
+    return map;
+}
```

Comment:
What is the reason to place this into it's own file?

Reply:
There will be ~250 entries in that map. I put it in a separate file so the other one is not cluttered.